Dockerfile.fat

```diff
@@ -1,18 +1,19 @@
 # Build the application
 FROM gradle:8.13-jdk21 AS build
 
-COPY build.gradle .
-COPY settings.gradle .
-COPY gradlew .
-COPY gradle gradle/
-RUN ./gradlew build -x spotlessApply -x spotlessCheck -x test -x sonarqube || return 0 
+# COPY build.gradle .
+# COPY settings.gradle .
+# COPY gradlew .
+# COPY gradle gradle/
 
 # Set the working directory
 WORKDIR /app
 
 # Copy the entire project to the working directory
 COPY . .
 
+RUN ./gradlew build -x spotlessApply -x spotlessCheck -x test -x sonarqube || return 0 
```

Comment:
I added this on line 8 designed to fail hence the return 0
Its done to grab and build the gradle files and dependences without needed the application code such that the docker cache point contains all gradle cache even if code changes are made,
Moving this to after COPY . . means it always is called with a fresh cache and redownloads it everytime

Reply:
Then we should define a default for `version.properties`

Reply:
including version.properties in that first COPY or to do a default sounds good to me
 or we hide the output of the first build since it truly doesn't effect the build right?

Reply:
@Frooodle Theoretically, it is not necessary to run `-x spotlessApply -x spotlessCheck -x test -x sonarqube` in the build stage, or am I wrong?